UI/Tests/DisplayAboutPageTest.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Core;
+using NUnit.Framework;
+
+namespace UI.Tests
+{
+    [TestFixture]
+    public class DisplayAboutPageTest : BaseTest
```

Comment:
As we have not a lot of tests we do not need a new files for it, but I agree that is better to please all tests in separate files as they are not in one functionality/suite

Reply:
I will move it to Tests then